README.md

```diff
@@ -456,7 +456,62 @@ This increasing the chances and the probability a lot that the automatic install
 A New setting is introduced - `EXHORT_PYTHON_INSTALL_BEST_EFFORTS` (as both env variable/key in `options` object)
 1. `EXHORT_PYTHON_INSTALL_BEST_EFFORTS`="false" - install requirements.txt while respecting declared versions for all packages.
 2. `EXHORT_PYTHON_INSTALL_BEST_EFFORTS`="true" - install all packages from requirements.txt, not respecting the declared version, but trying to install a version tailored for the used python version, when using this setting,you must set setting `MATCH_MANIFEST_VERSIONS`="false"
- 
+
+### Image Support 
+
+Generate vulnerability analysis report for container images.
+
+#### Code Example
+```java
+package com.redhat.exhort;
+
+import com.redhat.exhort.api.AnalysisReport;
+import com.redhat.exhort.image.ImageRef;
+import com.redhat.exhort.impl.ExhortApi;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+public class ExhortImageExample {
+
+    public static void main(String[] args) throws Exception {
+        // instantiate the Exhort API implementation
+        var exhortApi = new ExhortApi();
+
+        // create a reference to image test1 by specifying image name and its platform when applicable
+        var imageRef1 = new ImageRef("quay.io/test/test1:latest", "linux/amd64");
+
+        // create a reference to image test2 by specifying image name
+        var imageRef2 = new ImageRef("quay.io/test/test2:latest", null);
+
+        // get a byte array future holding a html Image Analysis reports
+        CompletableFuture<byte[]> htmlImageReport = exhortApi.imageAnalysisHtml(Set.of(imageRef1, imageRef2));
+
+        // get a map of AnalysisReport future holding a deserialized Image Analysis reports
+        CompletableFuture<Map<ImageRef, AnalysisReport>> imageReport = exhortApi.imageAnalysis(Set.of(imageRef1, imageRef2));
+    }
+}
+```
+
+#### Customization
+Customizing image analysis optionally by using *Environment Variables* or *Java Properties*.
+
+| Env / Property                | Description                                                                             | Default Value                                   |
+|-------------------------------|-----------------------------------------------------------------------------------------|-------------------------------------------------|
+| EXHORT_SYFT_PATH              | Custom path to the `syft` executable                                                    | syft                                            |
+| EXHORT_SYFT_CONFIG_PATH       | Custom path to the `syft` configuration file                                            | .syft.yaml, .syft/config.yaml, $HOME/.syft.yaml |
+| EXHORT_SYFT_IMAGE_SOURCE      | Image source for `syft`                                                                 |                                                 |
```

Comment:
@xieshenzh It's not that clear what is the purpose of this env var/property.
Maybe we can describe it as ` container images cli source for syft`, just as an initial baseline, the phrasing of it can be improved off course
In addition, please specify in the description what are the possible values for it , for example:
[ "podman" , "docker", "skopeo" ].